README.md

```diff
@@ -1,7 +1,18 @@
 # The Active Admin Store
-[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)           ](https://travis-ci.org/activeadmin/activeadmin)
-[![Dependency Status](https://gemnasium.com/badges/github.com/activeadmin/demo.activeadmin.info.svg)](https://gemnasium.com/github.com/activeadmin/demo.activeadmin.info)
+[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)](https://travis-ci.org/activeadmin/activeadmin)
 
 This is a test application to demo Active Admin.
 
 http://demo.activeadmin.info
+
+
+## Local development setup
+
+Dependencies: Ruby 2.5+ and PostgreSQL
+
+1. Clone project using Git
+2. Run `$ bundle install`
+3. Change Postgres username/password in `config/database.yml` (it will be good if this user has right to create DB)
+4. Run command `$ bin/rails db:setup` to create database with seed data.
+5. Start server: `$ bin/rails s`
+6. Navigate in browser to: http://localhost:3000/admin
```

Comment:
I think maybe better use `*` rather than numbers, so we don't need to readjust all the numbering everytime we add a step?